### ISSUE TITLE
Investigate the fillbiome command to see if it is possible to update the biomes of Chunks as they are built

### ISSUE THREAD

**Issue by AllYourBlocks** [AllYourBlocks/ChunkWars#163]
@slipcor Is it possible to use the new fillbiome command to correctly reflect the local biomes of Chunks as we paste them in Random mode?

(Players would likely need to log out and back in, or reload their rendered area or texture pack, in order to see the update visually update, because Minecraft)

From https://www.minecraft.net/en-us/article/minecraft-snapshot-22w46a

**COMMANDS**

- New command: fillbiome
- New execute sub-command: execute if|unless biome

**FILLBIOME**

Changes biome entries for an area. Note that biomes are not stored per-block, so affected positions may not match input precisely.

Syntax:

    fillbiome <from> <to> <biome>

Parameters:

- from: One corner of the area to fill
- to: The other corner of the area to fill
- biome: The biome to set

**EXECUTE**

Execute commands can now be conditional on biomes.

    Syntax: execute if|unless biome <pos> <biome>

**Comment by slipcor** [comment]
Yes. What are the biomes to use, apart from the ones that are obvious because they are in the chunk name?

Here are my placeholders so far

* bastion [basalt_deltas]
* fortress [nether_wastes]
* mansion [dark_forest]
* mountain [windswept_hills]
* nether [nether_wastes]
* pillager [plains]
* village [taiga]

**Comment by AllYourBlocks** [comment]
This is the current setup

| Chunk            | Biome                                           |
|------------------|-------------------------------------------------|
| Home             | Sunflower Plains                                |
| Village          | Taiga                                           |
| Desert           | Desert                                          |
| Jungle           | Jungle                                          |
| Ocean            | Ocean                                           |
| Ice Spikes       | Snowy Plains (might change to multiple biomes)  |
| Swamp            | Swamp (might change to multiple biomes)         |
| Flower Forest    | Flower Forest (likely changing to Cherry Grove) |
| Mushroom         | Mushroom Fields                                 |
| Badlands         | Eroded Badlands                                 |
| Outpost          | Savanna                                         |
| Mountain         | Jagged Peaks                                    |
| Nether           | Nether Wastes (might change to multiple biomes) |
| Woodland Mansion | Dark Forest                                     |
| Nether Fortress  | Nether Wastes                                   |
| Bastion          | Nether Wastes                                   |
